Engineering.md

```diff
@@ -108,3 +108,18 @@
 - I am actively involved in the retrospective process, and I am comfortable challenging and coaching my peers.
 - I am mentoring another engineer, deliberately helping them to round out their skill set.
 - I am sharing my on-the-job learning and experiences with others so they can be more effective in their roles.
+
+## Quality Assurance Engineer
+
+> I work with Octopus team members to improve the overall quality of the Octopus ecosystem. 
+
+### I ensure Octopus can ship with confidence
+
+- I contribute and provide feedback to engineering efforts so that the product is user-friendly, efficient, consistent, robust and meets expectations. 
```

Comment:
```suggestion
- I contribute and provide feedback to product, design, and engineering activities so that the product is user-friendly, efficient, consistent, robust, and useful to customers.
```